posting.d

```diff
@@ -237,6 +249,26 @@ final class PostProcess
 		{
 			log("Checking for spam");
 			status = PostingStatus.spamCheck;
+			if(this.userID) {
+				version(ProbablyNotAGoodIdea) {
+					immutable int spam_threshold = 1000;
+					foreach(int res; query!"SELECT spamchecks FROM Users WHERE id = ?"
```

Comment:
There is no `id` field in the `Users` table (because I'm not very good at designing databases).